pkg/client/client_test.go

```diff
@@ -17,34 +17,46 @@ package client
 import (
 	"testing"
 
+	client_testing "k8s.io/client-go/testing"
+
 	"gotest.tools/assert"
+	"k8s.io/apimachinery/pkg/runtime"
 	v1alpha1 "knative.dev/eventing-kafka/pkg/apis/sources/v1alpha1"
+	"knative.dev/eventing-kafka/pkg/client/clientset/versioned/typed/sources/v1alpha1/fake"
+	"knative.dev/kn-plugin-source-kafka/pkg/types"
 )
 
+var testNamespace = "fake-namespace"
+
+func setup() types.KafkaSourceClient {
```

Comment:
It might be a bit cleaner to return both `FakeSourcesV1alpha1, KafkaSourceCLient` as we have it in `knative/client` tests.
E.g.: https://github.com/knative/client/blob/master/pkg/sources/v1alpha2/ping_client_test.go#L30-L35